crates/nargo_cli/src/backends.rs

```diff
@@ -1 +1,24 @@
+use std::path::PathBuf;
+
+use acvm_backend_barretenberg::backends_directory;
 pub(crate) use acvm_backend_barretenberg::Backend;
+
+fn active_backend_file_path() -> PathBuf {
+    backends_directory().join(".selected_backend")
+}
+
+pub(crate) fn set_active_backend(backend_name: &str) {
+    std::fs::write(active_backend_file_path(), backend_name.as_bytes()).unwrap();
+}
+
+pub(crate) fn get_active_backend() -> String {
+    let active_backend_file = active_backend_file_path();
+
+    if !active_backend_file.is_file() {
+        let barretenberg = "acvm-backend-barretenberg";
```

Comment:
any reason to not make this string a constant in acvm-backend-barretenberg and pull it in from there?

Reply:
Hmm might cause a circular dependency in the future because ideally `BACKENDS_DIR` lives in nargo_cli or whatever will be the proxy for making proofs

Reply:
The `acvm-backend-barretenberg` crate is not long for this world anyway tbh.